mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -0,0 +1,208 @@
+			}
+		}
+
+		if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
+			if err := utils.DownloadFile(utils.DockerComposeFile, fileURL); err != nil {
+				return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to download %s file from %s", utils.DockerComposeFile, fileURL)))
+			}
+		}
+
+		//////// FLAGS
+		// Control whether to pull for new Meshery container images
+		if skipUpdateFlag {
+			log.Info("Skipping Meshery update...")
+		} else {
+			err := updateMesheryContainers()
+			if err != nil {
+				return errors.Wrap(err, utils.SystemError("failed to update meshery containers"))
+			}
+		}
+
+		// Reset Meshery config file to default settings
+		if utils.ResetFlag {
+			err := resetMesheryConfig()
+			if err != nil {
+				return errors.Wrap(err, utils.SystemError("failed to reset meshery config"))
+			}
+		}
+
+		log.Info("Starting Meshery again ...")
+		start := exec.Command("docker-compose", "-f", utils.DockerComposeFile, "up", "-d")
+		start.Stdout = os.Stdout
+		start.Stderr = os.Stderr
+
+		if err := start.Run(); err != nil {
+			return errors.Wrap(err, utils.SystemError("failed to run meshery server"))
+		}
+		checkFlag := 0 //flag to check
+
+		//connection to docker-client
+		cli, err := client.NewEnvClient()
+		if err != nil {
+			return errors.Wrap(err, utils.SystemError("failed to create new env client"))
+		}
+
+		containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{})
+		if err != nil {
+			return errors.Wrap(err, utils.SystemError("failed to fetch the list of containers"))
+		}
+
+		//check for container meshery_meshery_1 running status
+		for _, container := range containers {
+			if container.Names[0] == "/meshery_meshery_1" {
+				log.Info("Opening Meshery in your browser. If Meshery does not open, please point your browser to http://localhost:9081 to access Meshery.")
+
+				//check for os of host machine
+				if runtime.GOOS == "windows" {
+					// Meshery running on Windows host
+					err = exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Start()
+					if err != nil {
+						return errors.Wrap(err, utils.SystemError("failed to exec command"))
+					}
+				} else if runtime.GOOS == "linux" {
+					// Meshery running on Linux host
+					_, err := exec.LookPath("xdg-open")
+					if err != nil {
+						break
+					}
+					err = exec.Command("xdg-open", url).Start()
+					if err != nil {
+						return errors.Wrap(err, utils.SystemError("failed to exec command"))
+					}
+				} else {
+					// Assume Meshery running on MacOS host
+					err = exec.Command("open", url).Start()
+					if err != nil {
+						return errors.Wrap(err, utils.SystemError("failed to exec command"))
+					}
+				}
+
+				//check flag to check successful deployment
+				checkFlag = 0
+				break
+			} else {
+				checkFlag = 1
+			}
+		}
+
+		//if meshery_meshery_1 failed to start showing logs
+		//code for logs
```

Comment:
@subhamkrai Can be not write a separate function for logs and then call it here?
It would increase the code readability.